### PR TITLE
[MM-9785] Fix "add user to channel" from RHS when viewing different channel on center

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix "add user to channel" from RHS when viewing different channel on center

#### Ticket Link
Jira ticket: [MM-9785](https://mattermost.atlassian.net/browse/MM-9785)

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)

### CODE REVIEW

File: components/post_view/post_add_channel_member/post_add_channel_member.jsx [hunk 1]
```diff
@@ -66,19 +65,17 @@ export default class PostAddChannelMember extends React.PureComponent {
...
-                this.props.actions.addChannelMember(channel.id, userId);
-                sendAddToChannelEphemeralPost(currentUser, usernames[index], channel.id, post.root_id);
+                this.props.actions.addChannelMember(post.channel_id, userId);
+                sendAddToChannelEphemeralPost(currentUser, usernames[index], post.channel_id, post.root_id);
             });
 
             this.props.actions.removePost(post);
         }
-
-        browserHistory.push(`/${team.name}/channels/${channel.name}`);
```
Comment (saturninoabril) [inline components/post_view/post_add_channel_member/post_add_channel_member.jsx]: There's no point to redirect to the channel where at-mentioned user is added, so user who adds a user will remain on his currently viewing channel.

Comment (jespino) [reply components/post_view/post_add_channel_member/post_add_channel_member.jsx]: After removing that line team property is not longer needed. and from the channel, is only needed the channelType, maybe make sense to only receive the channelType.

Comment (saturninoabril) [reply components/post_view/post_add_channel_member/post_add_channel_member.jsx]: Make sense, I forgot to clean it up. Thanks!

File: components/post_view/post_add_channel_member/index.js [hunk 2]
```diff
@@ -7,18 +7,21 @@ import {addChannelMember} from 'mattermost-redux/actions/channels';
...
 import {getPost} from 'mattermost-redux/selectors/entities/posts';
 import {getChannel, getCurrentChannelId} from 'mattermost-redux/selectors/entities/channels';
-import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
 import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
 
 import PostAddChannelMember from './post_add_channel_member.jsx';
 
 function mapStateToProps(state, ownProps) {
     const currentChannelId = getCurrentChannelId(state);
+    const channel = getChannel(state, currentChannelId);
```
Comment (jespino) [inline components/post_view/post_add_channel_member/index.js]: We can use here directly the getCurrentChannel selector

**Comment by saturninoabril** [comment]
Thanks @jespino! I realized that I should be using the channel ID based from ephemeral post and not on the currently viewing channel.

**Pull request merged**